SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -205,24 +205,27 @@ void USpatialConnectionManager::ProcessLoginTokensResponse(const Worker_Alpha_Lo
 		return;
 	}
 
-	const FString& DeploymentToConnect = DevAuthConfig.Deployment;
+	FString DeploymentToConnect = DevAuthConfig.Deployment;
```

Comment:
out of curiosity: why did we make it not const anymore?

Reply:
It's modified in the `if (DeploymentToConnect.IsEmpty())` 
`DeploymentToConnect = UTF8_TO_TCHAR(LoginTokens->login_tokens[0].deployment_name);`

Reply:
Yeah I figured we can assign this to the deployment we choose when you don't have one specified.